src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Helpers/ShellCommand.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Helpers;
+
+public static class ShellCommand
+{
+    public delegate bool EnumThreadDelegate(IntPtr hwnd, IntPtr lParam);
+
+    private static bool containsSecurityWindow;
+
+    public static Process RunAsDifferentUser(ProcessStartInfo processStartInfo)
```

Comment:
Is this used anywhere?

(is most of this file used anywhere?)

Reply:
😅 - uh oh.

Reply:
Ummm i have literally no clue how I got here. brb.